models/migrations/v15.go

```diff
@@ -6,6 +6,7 @@ package migrations
 
 import (
 	"fmt"
+	"time"
```

Comment:
This file name should be renamed to v16.go since v15.go has been used by another PR just merged.